frontend/src/components/Forms/DevPortfolioForm/DevPortfolioForm.tsx

```diff
@@ -93,6 +94,11 @@ const DevPortfolioForm: React.FC = () => {
         headerMsg: 'Paragraph Submission Empty',
         contentMsg: 'Please write something for the paragraph section of the assignment.'
       });
+    } else if (!documentationText) {
+      Emitters.generalError.emit({
+        headerMsg: 'Documentation Empty',
+        contentMsg: 'Please write something for the documentation section of the assignment.'
```

Comment:
Thanks for these awesome changes Andrew (so quickly as well) I was wondering if you could make this more clarifying so that developers know that they can specify if the documentation is located within a submitted PR or a separate PR.

Reply:
No problem. Fully agreed on that suggestion. I've changed it to say:

> Please provide a link to at least one piece of documentation you added/updated. If it's
> included in the PRs you added above, you may simply write "Documentation located in PR (insert
> PR number here)". If you made a separate PR updating documentation in the codebase, please
> link that PR here.

Reply:
Awesome! Thanks Andrew!